src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -87,28 +89,57 @@ if (__DEV__) {
       // We were unable to guess which prop the user intended.
       // It is likely that the user was just blindly spreading/forwarding props
       // Components should be careful to only render valid props/attributes.
-      warning(
-        false,
-        'Unknown prop `%s` on <%s> tag. Remove this prop from the element. ' +
-        'For details, see https://fb.me/react-unknown-prop%s',
-        name,
-        tagName,
-        ReactComponentTreeDevtool.getStackAddendumByID(debugID)
-      );
+      unknownProp = name;
     }
+    return unknownProp;
   };
 }
 
+var groupWarnings = function(props, tagName, debugID) {
```

Comment:
Let’s rename `props` to `unknownPropNames` to make it clear `props` is not a props object like everywhere else.